cmd/bucket-handlers.go

```diff
@@ -474,6 +474,16 @@ func (api objectAPIHandlers) DeleteMultipleObjectsHandler(w http.ResponseWriter,
 	versioned := globalBucketVersioningSys.Enabled(bucket)
 	suspended := globalBucketVersioningSys.Suspended(bucket)
 
+	purgeOnDelete := false
+	if d := r.Header.Get(xhttp.MinIOPurgeOnDelete); d != "" {
+		if b, err := strconv.ParseBool(d); err == nil {
```

Comment:
```suggestion
		if b, err := strconv.ParseBool(d); err == nil {
```
Lets only support `true, false` not `ParseBool`  - it is too generic.

Reply:
Actually now to think of this, relaxed behavior is okay. Let's keep it ParseBool for now.